pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1781,7 +1781,7 @@ func getPilotRandomSamplingEnv() float64 {
 	f := features.TraceSampling
 	if f < 0.0 || f > 100.0 {
 		log.Warnf("PILOT_TRACE_SAMPLING out of range: %v", f)
-		return 100.0
+		return 1.0
```

Comment:
nit:
How about declaring a constant and also using that in the test? 
This will save efforts next time we want to update `TraceSampling` to a different value.

Reply:
That is a good idea.  Could that be a follow up PR?  I could see this for 100.0 as a constant as well.